integration/daemon/daemon_linux_test.go

```diff
@@ -173,15 +172,8 @@ func TestDaemonDefaultBridgeIPAM_Docker0(t *testing.T) {
 			// The bridge's address/subnet should be ignored, this is a change
 			// of fixed-cidr.
 			expIPAMConfig: []network.IPAMConfig{
```

Comment:
The changes in `daemon/network.go` cause the IPAM config to be updated to reflect the actual allocated network data which was causing these integration tests to fail (they expected to report only the initial IPAM config which didn't include the allocated gateway information). This means that on first run inspected networks should correctly reflect the allocated state vs. requiring a restart for the config to accurately reflect the allocated data as the old comment indicated.